app/templates/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "<%= projectSlug %>",
+  "description": "<%= projectName %> project files",
+  "version": "1.0.0",
+  "browserslist": [
+    "defaults"
+  ],
+  "scripts": {
+    "serve": "gulp serve",
+    "build": "gulp build",
+    "dev": "gulp dev",
+    "postinstall": "npm run build"
+  },
+  "dependencies": {
+    "@babel/polyfill": "^7.2.5",
+    "conditioner-core": "^2.3.1",
+    "element-closest": "^3.0.1",
+    "focus-visible": "^5.1.0",
+    "intersection-observer": "^0.9.0",
+    "supple": "^3.0.0",
+    "svgxuse": "^1.2.6"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.9.0",
+    "@babel/preset-env": "^7.9.5",
+    "@pattern-lab/core": "^5.7.0",
+    "@pattern-lab/engine-mustache": "^5.0.0",
+    "@pattern-lab/uikit-bare": "0.0.1",
+    "autoprefixer": "^9.7.6",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.1.0",
+    "babel-plugin-dynamic-import-webpack": "^1.1.0",
+    "babel-plugin-import": "^1.13.0",
+    "babel-plugin-syntax-dynamic-import": "^6.18.0",
+    "browser-sync": "^2.26.7",
+    "cssnano": "^4.1.8",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-loader": "^4.0.0",
+    "eslint-plugin-import": "^2.20.2",
+    "gulp": "^4.0.2",
+    "gulp-dart-sass": "^1.0.1",
+    "gulp-eslint": "^6.0.0",
+    "gulp-notify": "^3.2.0",
+    "gulp-plumber": "^1.2.1",
+    "gulp-postcss": "^8.0.0",
+    "gulp-sourcemaps": "^2.6.5",
+    "gulp-stylelint": "^13.0.0",
+    "gulp-svg-sprite": "^1.5.0",
+    "node-sass-tilde-importer": "^1.0.2",
+    "postcss-normalize": "^9.0.0",
+    "stylelint": "^13.3.2",
+    "stylelint-config-standard": "^20.0.0",
```

Comment:
ik zou toch adviseren om `stylelint-config-supple` te gebruiken. Deze is specifieker geschreven voor de manier waarop supple geschreven is en de ITCSS manier die wij gebruiken. Als daar nog dingen in zitten die je mist of anders wil zien: be my guest en maak een PR 😉